libs/components/editor-core/src/kanban/atom.ts

```diff
@@ -96,40 +106,42 @@ export const calcCardGroup = (
 /**
  * Set group value for the card block
  */
-export const moveCardToGroup = async (
-    groupById: RecastPropertyId,
-    cardBlock: RecastItem,
-    group: KanbanGroup
-) => {
-    const { setValue, removeValue } = getRecastItemValue(cardBlock);
+export const moveCardToGroup = async ({
```

Comment:
Please revert the modification.

The pendant and the kanban depend on the `recastBlock`,

the pendant and the kanban should be independent,

cross-referencing is very dangerous.